packages/unchained-client/src/evm/optimism/parser/index.ts

```diff
@@ -2,10 +2,15 @@ import { optimismAssetId } from '@shapeshiftoss/caip'
 
 import { Tx } from '../../../generated/optimism'
 import { BaseTransactionParser, TransactionParserArgs } from '../../parser'
+import * as zrx from '../../parser/zrx'
+
+export const ZRX_PROXY_CONTRACT = '0xDEF1ABE32c034e558Cdd535791643C58a13aCC10'
```

Comment:
https://optimistic.etherscan.io/address/0xdef1abe32c034e558cdd535791643c58a13acc10

Reply:
any merit to renaming this to `ZRX_OPTIMISM_PROXY_CONTRACT` to avoid this being exported/reused with wrong chain?

Reply:
I was only exporting to use in tests and figured since it was in the optimism unchained client namespace this was sufficient (I personally don't like redundant naming prefixes to namespaced packages), but I can make explicit just for extra safety

Reply:
https://github.com/shapeshift/lib/pull/1178/commits/20bf09cd6747cccd9fef800cea1d7471acb83b7e